src/plugins/intel_gpu/include/intel_gpu/plugin/compiled_model.hpp

```diff
@@ -27,11 +27,13 @@ class CompiledModel : public ov::ICompiledModel {
     CompiledModel(std::shared_ptr<ov::Model> model,
                   const std::shared_ptr<const ov::IPlugin>& plugin,
                   RemoteContextImpl::Ptr context,
-                  const ExecutionConfig& config);
+                  const ExecutionConfig& config,
+                  const bool loaded_from_cache = false);
     CompiledModel(cldnn::BinaryInputBuffer& ib,
                   const std::shared_ptr<const ov::IPlugin>& plugin,
                   RemoteContextImpl::Ptr context,
-                  const ExecutionConfig& config);
+                  const ExecutionConfig& config,
+                  const bool loaded_from_cache = false);
```

Comment:
do we need to read value from `update_config` passed to from `Core` ?

Reply:
The code missed to upload :(